src/webgpu/shader/execution/builtin/isinf.spec.ts

```diff
@@ -0,0 +1,65 @@
+export const description = `WGSL execution test. Section: Value-testing built-in functions Function: isInf`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { GPUTest } from '../../../gpu_test.js';
+import { TypeBool, TypeF32, bool, f32, f32Bits } from '../../../util/conversion.js';
+import { subnormalF32Examples, normalF32Examples } from '../../values.js';
+
+import { anyOf, run, kBit } from './builtin.js';
+
+export const g = makeTestGroup(GPUTest);
+
+g.test('value_testing_builtin_functions,isInf')
+  .uniqueId('3591ae3f3daa3871')
+  .specURL('https://www.w3.org/TR/2021/WD-WGSL-20210929/#value-testing-builtin-functions')
+  .desc(
+    `
+isInf:
+isInf(e: I ) -> T Test for infinity according to IEEE-754. Component-wise when I is a vector. (OpIsInf)
+
+Please read the following guidelines before contributing:
+https://github.com/gpuweb/cts/blob/main/docs/plan_autogen.md
+`
+  )
+  .params(
+    u =>
+      u
+        .combine('storageClass', ['storage_r'] as const)
+        .combine('vectorize', [undefined, 2] as const)
+    //.combine('vectorize', [undefined, 2, 3, 4] as const)
+  )
+  .fn(async t => {
+    const make = (the_input: number, expected: boolean) => {
+      return {
+        input: f32(the_input),
+        expected: bool(expected),
+      };
+    };
+    const anyBool = anyOf(bool(false), bool(true));
+    const cases = [
+      // Non-infinity
+      make(0.0, false),
```

Comment:
nit: there's an odd mix of `make` and inlined cases here. Is `make` that useful?